conda/dali_bindings/recipe/meta.yaml

```diff
@@ -0,0 +1,103 @@
+# (C) Copyright IBM Corp. 2019. All Rights Reserved.
+# Copyright (c) 2019-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package:
+  name: nvidia-dali-cuda{{ environ.get('CUDA_VERSION', '') | replace(".","") }}
+  version: {{ environ.get('DALI_CONDA_BUILD_VERSION', '') }}
+
+source:
+  # Beware: Only committed files are used
+  - git_url: ../../..
+
+build:
+  script_env:
+   - CUDA_TARGET_ARCHS
+   - CMAKE_BUILD_TYPE
+   - BUILD_TEST
+   - BUILD_BENCHMARK
+   - BUILD_FUZZING
+   - BUILD_NVTX
+   - BUILD_LMDB
+   - BUILD_JPEG_TURBO
+   - BUILD_NVJPEG
+   - BUILD_LIBTIFF
+   - BUILD_LIBSND
+   - BUILD_LIBTAR
+   - BUILD_FFTS
+   - BUILD_CFITSIO
+   - BUILD_NVOF
+   - BUILD_NVDEC
+   - BUILD_NVML
+   - WITH_DYNAMIC_CUDA_TOOLKIT
+   - WITH_DYNAMIC_NVJPEG
+   - WITH_DYNAMIC_CUFFT
+   - WITH_DYNAMIC_NPP
+   - VERBOSE_LOGS
+   - WERROR
+   - BUILD_WITH_ASAN
+   - BUILD_WITH_LSAN
+   - BUILD_WITH_UBSAN
+   - NVIDIA_BUILD_ID
+   - GIT_SHA
+   - DALI_TIMESTAMP
+   - NVIDIA_DALI_BUILD_FLAVOR
+   - LD_LIBRARY_PATH
+   - DALI_CONDA_BUILD_VERSION
+   - CUDA_VERSION
+  string: py{{ python | replace(".","") }}_{{ environ.get('NVIDIA_BUILD_ID', '') }}
+
+requirements:
+  build:
+    skip: True  # [not linux]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - cmake >=3.12.4
+    - make
+    - patchelf
+    - python-clang
+    - git-lfs
+    - astunparse >=1.6.0
+    - gast >=0.3.3
+    - dm-tree >=0.1.8
+    - black =23.11.0
```

Comment:
Note: Other PR is about to update to 23.12.0

Reply:
Will rebase later.